vertical-healthcare_ch/forum_datenaustausch_ch.invoice_base/src/main/java/de/metas/vertical/healthcare_ch/forum_datenaustausch_ch/base/export/invoice/InvoiceImportClientImpl.java

```diff
@@ -82,27 +90,130 @@ private ImportedInvoiceResponse import0(@NonNull final ImportInvoiceResponseRequ
 		final String xsdName = XmlIntrospectionUtil.extractXsdValueOrNull(inputStreamToUse);
 		inputStreamToUse.reset();
 
-		final CrossVersionResponseConverter converter = crossVersionServiceRegistry.getResponseConverterForXsdName(xsdName);
+		final CrossVersionResponseConverter converter = crossVersionServiceRegistry.getResponseConverterForXsdName(Objects.requireNonNull(xsdName));
```

Comment:
nice, i wasn't aware of `requireNonNull`

Reply:
on the other hand, i think our own `Check.assumeNotNull`is better, because you can add an error message, yo a dev can see in the console wtf it is that was null